### DISCUSSION TITLE
When I deploy to cloud run , System show error "Step #0: npm error peer react@"^18 || ^19" from @tanstack/react-query@5.59.20" (Use with next.js15

### DISCUSSION THREAD

**Discussion by angpao** [TanStack/query#8280] [General]
Step #0: npm error Found: react@19.0.0-rc-66855b96-20241106
Step #0: npm error node_modules/react
Step #0: npm error   react@"19.0.0-rc-66855b96-20241106" from the root project
Step #0: npm error
Step #0: npm error Could not resolve dependency:
Step #0: npm error peer react@"^18 || ^19" from @tanstack/react-query@5.59.20
Step #0: npm error node_modules/@tanstack/react-query
Step #0: npm error   @tanstack/react-query@"^5.59.20" from the root project

**Comment by TkDodo** [comment]
it seems that `19.0.0-rc-66855b96-20241106` does not satisfy `^19` 🤷‍♂️ . Install with `--force` or `--legacy-peer-deps`.

**Reply by marklundin** [reply]
The `-rc` is a prerelease identifier which doesn't support `^19`
